docs/operations/storageclasses.md

```diff
@@ -122,3 +122,44 @@ parameters:
   csi.storage.k8s.io/secret-name: storageos-api
   csi.storage.k8s.io/secret-namespace: storageos
 ```
+
+### Topology volume Storage Class
```

Comment:
We might want to change this to `Allowed Topology volumes Storage Class`, to  differentiate it from `Topology Aware` a bit more

Reply:
Good idea, I will use that